src/test/resources/alma/(CKB)5280000000199164.json

```diff
@@ -0,0 +1,204 @@
+    "sublocation" : "https://hbz-network.userservices.exlibrisgroup.com/view/uresolver/49HBZ_NETWORK/openurl?u.ignore_date_coverage=true&rft.mms_id=9947420472406443"
+  } ],
+  "medium" : [ {
+    "id" : "http://rdaregistry.info/termList/RDAMediaType/1003",
+    "label" : "Datenträger"
+  }, {
+    "id" : "http://rdaregistry.info/termList/RDACarrierType/1018",
+    "label" : "Online-Ressource"
+  } ],
+  "type" : [ "Periodical", "BibliographicResource" ],
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "language" : [ {
+    "label" : "Deutsch",
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/ger"
+  } ],
+  "sameAs" : [ {
+    "id" : "http://worldcat.org/oclc/864921933",
+    "label" : "OCLC Ressource"
+  } ],
+  "extent" : "Online-Ressource",
+  "note" : [ "Gesehen am 29.09.2020" ],
+  "contribution" : [ {
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/ctb",
+      "label" : "Mitwirkende"
+    },
+    "type" : [ "Contribution" ],
+    "agent" : {
+      "gndIdentifier" : "5500538-X",
+      "id" : "https://d-nb.info/gnd/5500538-X",
+      "type" : [ "CorporateBody" ],
+      "label" : "Evangelisch-Lutherisches Missionswerk Leipzig"
+    }
+  } ],
+  "describedBy" : {
+    "sourceOrganization" : {
+      "id" : "http://lobid.org/organisations/DE-0355#!",
+      "label" : "lobid Organisation"
+    },
+    "provider" : {
+      "id" : "http://lobid.org/organisations/DE-DE-101#!",
+      "label" : "lobid Organisation"
+    },
+    "modifiedBy" : {
+      "id" : "http://lobid.org/organisations/DE-9001#!",
+      "label" : "lobid Organisation"
+    },
+    "dateModified" : "20210419",
+    "dateCreated" : "20210419",
+    "id" : "http://lobid.org/resources/ALMA99370682219806441",
+    "label" : "Webseite der hbz-Ressource ALMA99370682219806441",
+    "type" : [ "BibliographicDescription" ],
+    "inDataset" : {
+      "id" : "http://lobid.org/resources/dataset#!",
+      "label" : "lobid-resources – Der hbz-Verbundkatalog als Linked Open Data"
+    },
+    "resultOf" : {
+      "type" : [ "CreateAction" ],
+      "endTime" : "2021-05-31T15:27:52",
+      "instrument" : {
+        "id" : "https://github.com/hbz/lobid-resources",
+        "type" : [ "SoftwareApplication" ],
+        "label" : "Software lobid-resources"
+      },
+      "object" : {
+        "id" : "http://lobid.org/hbz01/ALMA99370682219806441",
+        "type" : [ "DataFeedItem" ],
+        "label" : "hbz-Ressource ALMA99370682219806441 im Exportformat MARC21 XML",
+        "inDataset" : {
+          "id" : "https://datahub.io/dataset/hbz_unioncatalog",
+          "label" : "hbz_unioncatalog"
+        }
+      }
+    },
+    "license" : [ {
+      "id" : "http://creativecommons.org/publicdomain/zero/1.0",
+      "label" : "Creative Commons-Lizenz CC0 1.0 Universal"
+    } ]
+  },
+  "natureOfContent" : [ {
+    "id" : "http://d-nb.info/gnd/4067488-5",
+    "label" : "Zeitschrift"
+  }, {
+    "label" : "Zeitschrift"
+  }, {
```

Comment:
Here we have duplications. I thought we have had already fixed that - or is this a new issue?

Reply:
It is a different issue. This is part of "natureOfContent". But this will be a tricky I think this is related to 
#1268 since the source data comes from 655 and 689 I don't know a easy way to unique the results. But please open another ticket.

Reply:
Ah, you are right, and I opened already https://github.com/hbz/lobid-resources/issues/1271 for it.